examples/AN00162_i2s_loopback_demo/doc/rst/AN00162.rst

```diff
@@ -22,32 +22,34 @@ library to create an |I2S| master digital loopback.
 Block Diagram
 .............
 
-.. figure:: images/block_diagram.*
-   :width: 80%
+.. figure:: images/block_diagram.drawio.*
+   :width: 60%
 
    Application block diagram
 
-The application fits entirely within one logical core. The |I2S|, GPIO,
-and |I2C| drivers are all time-multiplexed with the loopback application.
+The main application fits within one thread with a remote |I2C| task to allow
+the audio hardware to be configured remotely from the other tile. This is provided
+by the ``lib_board_support`` library which takes care of the audio hardware setup.
 
+The |I2S| task calls back to the ``i2s_loopback`` task and the processing in the
+``i2s_loopback`` task is performed in-between the I/O operations of |I2S|.
 
 |I2S| loopback demo
 -------------------
 
 The Makefile
 ............
 
-To start using the |I2S|, you need to add ``lib_i2s`` to your Makefile::
+To start using the |I2S|, you need to add ``lib_i2s`` to your CMake Makefile::
 
-  USED_MODULES = .. lib_i2s ...
+  set(APP_DEPENDENT_MODULES   "lib_i2s")
```

Comment:
This is potentially confusing because the CMakeLists.txt in this app note does not contain this line - it gets included from deps.cmake.

Reply:
I added an explicit line:

The dependancies for this example are specified by ``deps.cmake`` in the root of the ``lib_i2s`` repo
and are included in the application ``CMakeLists.txt`` file.